docs/UserGuide.md

```diff
@@ -905,4 +943,5 @@ KonTActs data are saved automatically as a JSON file `[JAR file location]/data/k
 3. **When using excel to edit telegram username**, typing "@" will trigger the command function which will output a "Function is not valid" error. The remedy is to prefix the telegram username with an apostrophe `'` to escape from the command function. For example: typing `'@james`.
 ![excel known issue](images/excelKnownIssue.png)
 4. **Certain fields such as email are not displayed on the list view**, and can only be seen via `view`. This is intended so that the list view does not become cluttered with too much information.
+5. Names that contains the `/` characters will cause konTActs to show an error message although a person can legally have `/` in their legal name such as `Kumar S/O Mahesh`. This is a limitation within konTActs as it uses the `/` charcter internally. The solution for this is to totally omit the `/` altogether. In this case, the name should be added as `Kumar SO Mahesh`
```

Comment:
This is just my suggestion: "..should be added as `Kumar SO Mahesh` or `Kumar son of Mahesh"